frontend/src/components/RecipeComments/RecipeComments.js

```diff
@@ -3,36 +3,66 @@ import map from 'lodash/map';
 import { createComment } from '../../lib/apiClient';
 import './RecipeComments.css';
 
-const RecipeComments = ({ comments, recipeId, user }) => {
+const RecipeComments = ({
+  comments, recipeId, user, reloadComments,
+}) => {
   const [comment, setComment] = useState('');
 
+  const formatMinutes = (minute) => {
+    const zero = 0;
```

Comment:
This seems unnecessary. You should just be about to do something like this on line 14:
```
return `0${minute}`;
```